README.md

```diff
@@ -135,6 +135,7 @@ To run as a desktop app:
 
    ```
    npm install electron
+   npm run install:electron
    node_modules/.bin/electron .
```

Comment:
Sounds like the correct way of doing this is with the electron-builder install-app-deps in postinstall (https://github.com/electron-userland/electron-builder/wiki/Two-package.json-Structure) (assuming the 2 package.json structure is what you were going for here)